pkg/server/config.go

```diff
@@ -16,21 +16,30 @@ package server
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/abcxyz/pkg/cli"
 )
 
 // HandlerConfig defines the set over environment variables required
 // for running this application.
 type HandlerConfig struct {
-	Port           string `env:"PORT,default=8080"`
-	ProjectID      string `env:"PROJECT_ID,required"`
-	SuccessTopicID string `env:"PMAP_SUCCESS_TOPIC_ID,required"`
-	FailureTopicID string `env:"PMAP_FAILURE_TOPIC_ID"`
+	Port                 string `env:"PORT,default=8080"`
+	ProjectID            string `env:"PROJECT_ID,required"`
+	SuccessTopicID       string `env:"PMAP_SUCCESS_TOPIC_ID,required"`
+	FailureTopicID       string `env:"PMAP_FAILURE_TOPIC_ID"`
+	DefaultResourceScope string `env:"PMAP_RESOURCE_SCOPE,required"`
```

Comment:
Please add comment specifying this field is only required for mapping service, and remove required. since the naming is changed, please change the env name accordingly.

Reply:
Added comments, I think we can keep the env name, as it matches `PMAP_FAILURE_TOPIC_ID`, and they are both only used in mapping service.

Reply:
To correct, PMAP_FAILURE_TOPIC_ID is optional for policy, there is a case when user customize a policyprocessor and want to send the failure event to failure table.